Scripts/World/Places/place.gd

```diff
@@ -9,23 +9,33 @@ extends Node
 @export var coord : Vector2i
 
 
-static func _inner_create_place(new_data_tile : DataTile) -> Place:
-		match new_data_tile.type:
-			"sawmill":
-				return Deposit.new(Goods.new(5,0,0), Goods.new(1,0,0))
-			"iron_mine":
-				return Deposit.new(Goods.new(0,5,0), Goods.new(0,1,0))
-			"ruby_cave":
-				return Deposit.new(Goods.new(0,0,5), Goods.new(0,0,1))
-			"elf_city", "orc_city":
-				return City.new()
-			_:#"sentinel", "wall", "empty"
-				return null
+static func _inner_create_place(new_data_tile : DataTile, coord : Vector2i) -> Place:
+	match new_data_tile.type:
+		# Resource Outposts
+		"sawmill":
+			return Deposit.new(Goods.new(5,0,0), Goods.new(1,0,0))
+		"iron_mine":
+			return Deposit.new(Goods.new(0,5,0), Goods.new(0,1,0))
+		"ruby_cave":
+			return Deposit.new(Goods.new(0,0,5), Goods.new(0,0,1))
+		
+		# city:
+		"elf_city", "orc_city":
+			return City.new()
+		"wood_hunt":
+			return HuntSpot.new(coord, CFG.HUNT_WOOD_PATH, [Goods.new(3,0,0), Goods.new(6,0,0)])
+		"iron_hunt":
+			return HuntSpot.new(coord, CFG.HUNT_IRON_PATH, [Goods.new(0,3,0), Goods.new(0,6,0)])
+		"ruby_hunt":
+			return HuntSpot.new(coord, CFG.HUNT_RUBY_PATH, [Goods.new(0,0,3), Goods.new(0,0,6)])
+
+		_:#"sentinel", "wall", "empty"
+			return null
 
 
 static func create_place(new_data_tile : DataTile, \
 		new_coord : Vector2i) -> Place:
-	var new_place = _inner_create_place(new_data_tile)
+	var new_place = _inner_create_place(new_data_tile, new_coord)
```

Comment:
no need for new_coord
it is set in code below
```
if new_place != null:
		new_place.coord = new_coord
```
as i see it is used in constructor for
`alive_army = WM.spawn_neutral_army(neutral_armies[0], coord)`
spawning army should not happen directly in constructor
there should be extra callback like "on_world_map_start"
so that creating a Place for tests does not change WM

Reply:
ideally WM would spawn units by inspecting the place so that place does not kneed to know anything about WM
but this is more complicated so its for future refactors